app/code/Magento/Reports/Block/Adminhtml/Bestsellers/Filter/Form.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Reports\Block\Adminhtml\Bestsellers\Filter;
+
+use Magento\Framework\Data\Form\Element\Fieldset;
+use \Magento\Reports\Block\Adminhtml\Filter\Form as ReportsBlockFilterForm;
+
+/**
+ * Sales Adminhtml report bestseller filter form
+ *
+ * @package Magento\Reports\Block\Adminhtml\Bestsellers\Filter
+ */
+class Form extends ReportsBlockFilterForm
+{
+    /**
+     * @inheritdoc
+     */
+    protected function _prepareForm()
+    {
+        parent::_prepareForm();
+        /** @var Fieldset $fieldset */
+        $fieldset = $this->getForm()->getElement('base_fieldset');
+        if (is_object($fieldset) && $fieldset instanceof Fieldset) {
```

Comment:
I believe `is_object` check is not needed there since we already have `instanceof` there

Reply:
Redundant condition was removed